src/containers/AccountModal/forms/Login.tsx

```diff
@@ -10,7 +10,11 @@ import { removeQueryParam } from '#src/utils/location';
 import type { LoginFormData } from '#types/account';
 import { login } from '#src/stores/AccountController';
 
-const Login = () => {
+type Props = {
+  message?: string;
```

Comment:
It might be nice to rename this prop to `messageKey` or `messageParam` or something to make it more clear that it's not the actual string that will be displayed, rather a lookup key. You can rename here, in LoginForm, and in AccountModal.

Reply:
892317022b80decbf8cb56b9749447673a899c88